pkg/controller/nodes.go

```diff
@@ -251,16 +251,11 @@ func (cont *AciController) nodeChanged(obj interface{}) {
 		cont.nodePodNetCache[node.ObjectMeta.Name] = nodePodNet
 	}
 
-	netval, netok :=
-		node.ObjectMeta.Annotations[metadata.PodNetworkRangeAnnotation]
-	if netok {
-		if netval != nodePodNet.podNetIpsAnnotation {
-			cont.mergePodNet(nodePodNet, netval, logger)
-		}
-	}
+	netval := node.ObjectMeta.Annotations[metadata.PodNetworkRangeAnnotation]
```

Comment:
since its a string map netok should have been true if the annotation was present ? I guess the actual change is that we no longer do the mergePodNet. I need to look at what it does but if you can explain how its causing the overlap issue it would be helpful.